src/components/DateInput.jsx

```diff
@@ -234,7 +240,8 @@ class DateInput extends React.PureComponent {
           disabled={disabled}
           readOnly={typeof readOnly === 'boolean' ? readOnly : isTouch}
           required={required}
-          aria-describedby={screenReaderMessage && screenReaderMessageId}
+          aria-describedby={ariaDescribedby || (screenReaderMessage && screenReaderMessageId)}
```

Comment:
There may be cases where someone may want to have both `screenReaderMessage` and `ariaDescribedBy` set. This creates a bit of a complication as to the order, since it's hard to know what order to put them without knowing what content is there. Alternatively, it might be good to make the two props mutually exclusive so that consumers know that they can only use one.

Reply:
Which direction do you think is best?

Reply:
Perhaps for simplicity, they can be mutually exclusive. If someone wants to use `ariaDescribedby` and have another screen reader message, they could do that manually and pass in the related `id` with `ariaDescribedby`.